### PR TITLE
TEN-1451/24.10.0/invalid email error tests

### PR DESCRIPTION
**Pull request by markbainix**
Ten 1451 invalid email errors

### CODE REVIEW

**Comment by ericbsd** [comment]
image

File: test_cases/webui/local_users/test_local_user_errors.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        Test Steps:
+        1. Click Add User button
+        2. Edit User email
+        3. Verify error message
+        """
+        LU.click_add_user_button()
+
+        # Verify no @ or . or com email "test"
+        LU.set_user_email('test')
+        assert COM.is_text_visible('Value must be a valid email address') is True
```
[the remark this answers is not in the document]
Comment (markbainix) [reply test_cases/webui/local_users/test_local_user_errors.py]: added

File: test_cases/webui/local_users/test_local_user_errors.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    def teardown_test(self):
+        """
+        This method clears any test users after test is run for a clean environment
+        """
+        yield
+        # Clean up environment.
+
+    @allure.tag("Create")
+    @allure.story("Local User Invalid Email Errors")
+    def test_local_user_invalid_email_errors(self) -> None:
```
[the remark this answers is not in the document]
Comment (markbainix) [reply test_cases/webui/local_users/test_local_user_errors.py]: refactored

**Review by ericbsd — APPROVED** [review]
LGTM

**Pull request merged**